workflows/tomte.nf

```diff
@@ -120,8 +130,8 @@ workflow TOMTE {
     ch_chrom_sizes           = ch_references.chrom_sizes
     ch_sequence_dict         = params.sequence_dict           ? Channel.fromPath(params.sequence_dict).collect()
                                                               : ( ch_references.sequence_dict            ?: Channel.empty() )
-    ch_genome_fai            = params.fasta_fai               ? Channel.fromPath(params.fasta_fai).collect()
-                                                              : ( ch_references.fasta_fai                ?: Channel.empty() )
+    //ch_genome_fai            = params.fai                     ? Channel.fromPath(params.fai).collect()
+    //                                                          : ( ch_references.fai                ?: Channel.empty() )
```

Comment:
remove or comment back in?

Reply:
Sorry about that, I forgot to remove them. They should go out, otherwise it doesn't work to do this: `(!star_index)` to check if a channel is empty